text/0976-glint-as-default.md

```diff
@@ -0,0 +1,161 @@
+---
+stage: accepted
+start-date: 2023-09-29T00:00:00.000Z
+release-date: # In format YYYY-MM-DDT00:00:00.000Z
+release-versions:
+teams: # delete teams that aren't relevant
+  - cli
+  - framework
+  - learning
+  - typescript
+prs:
+  accepted: https://github.com/emberjs/rfcs/pull/976
+project-link:
+suite:
+---
+
+# Enable Glint by Default
+
+## Summary
+
+[Glint](https://typed-ember.gitbook.io/glint) should be enabled as a default for Ember projects.
+
+This RFC proposes adopting Glint as a default/recommended tool in the Ember.js framework. Glint is a static TypeScript-based template type checker that aims to improve developer experience and catch template-related errors at build time. By incorporating Glint into Ember.js, we can enhance type safety, provide better tooling, and encourage best practices when working with templates.
+
+Adopting Glint as a default/recommended tool in Ember.js is a step towards improving developer experience, reducing errors, and promoting best practices in Ember.js template development. By integrating Glint and providing guidance, Ember.js can stay at the forefront of modern web development practices.
```

Comment:
This sentence seems like a clear duplicate of the previous sentence and does not seem necessary.

- is a step towards improving developer experience
  - that aims to improve developer experience

- reducing errors
  - catch template-related errors

- promoting best practices
  - encourage best practices when working with templates